plugin.json

```
@@ -19,12 +19,20 @@
         "help_text": "If set the plugin will also filter posts containing plain text links in addition to filtering embedded text links."
       },
       {
-        "key": "WarningMessage",
-        "display_name": "Warning Message:",
+        "key": "NewPostWarningMessage",
```

Propose we use `create`/`edit` or `new`/`existing` and not a combination of both.
In this case, I think `Create` makes sense.

```suggestion
        "key": "CreatePostWarningMessage",
```